e2e/specs/confirmations-redesigned/transactions/gas-fee-tokens-eip-7702.spec.ts

```diff
@@ -0,0 +1,243 @@
+      id: 'faaab4c5-edf5-4077-ac75-8d26278ca2c5',
+    },
+  },
+};
+
+describe(
+  SmokeConfirmationsRedesigned('Send native asset using EIP-7702'),
+  () => {
+    const testSpecificMock = async (
+      mockServer: Mockttp,
+      options: { success: boolean } = { success: true },
+    ) => {
+      const { success } = options;
+      await setupMockRequest(mockServer, {
+        requestMethod: 'GET',
+        url: SIMULATION_ENABLED_NETWORKS_WITH_RELAY.urlEndpoint,
+        response: SIMULATION_ENABLED_NETWORKS_WITH_RELAY.response,
+        responseCode: 200,
+      });
+      await setupMockRequest(mockServer, {
+        requestMethod: 'GET',
+        url: `${LOCALHOST_SENTINEL_URL}/network`,
+        response: SIMULATION_ENABLED_NETWORKS_WITH_RELAY.response,
+        responseCode: 200,
+      });
+      await setupMockRequest(mockServer, {
+        ...SIMULATION_GAS_STATION_MOCK,
+        requestMethod: 'POST',
+        url: `${LOCALHOST_SENTINEL_URL}/`,
+        response: SIMULATION_GAS_STATION_MOCK.response,
+        responseCode: 200,
+      });
+      await setupRemoteFeatureFlagsMock(
+        mockServer,
+        Object.assign({}, ...remoteFeatureEip7702),
+      );
+      await setupMockRequest(mockServer, {
+        requestMethod: 'GET',
+        url: `${TRANSACTION_RELAY_STATUS_NETWORKS_MOCK.urlEndpoint}`,
+        response: {
+          transactions: [
+            {
+              hash: TRANSACTION_RELAY_STATUS_NETWORKS_MOCK.response
+                .transactions[0].hash,
+              status: success ? RelayStatus.Success : 'FAILED',
+            },
+          ],
+        },
+        responseCode: 200,
+      });
+    };
+
+    const fixtureBase = {
+      fixture: new FixtureBuilder()
+        .withGanacheNetwork('0x1')
+        .withDisabledSmartTransactions()
+        .build(),
+      restartDevice: true,
+      localNodeOptions: [
+        {
+          type: LocalNodeType.anvil,
+          options: {
+            hardfork: 'prague' as Hardfork,
+            loadState:
+              './e2e/specs/confirmations-redesigned/transactions/7702/withUpgradedAccount.json',
+          },
+        },
+      ],
+      testSpecificMock,
+    };
+
+    it('should send ETH via EIP-7702', async () => {
+      await withFixtures(fixtureBase, async () => {
+        await loginToApp();
+        await device.disableSynchronization();
+        await WalletView.tapWalletSendButton();
+        await SendView.selectEthereumToken();
+        await SendView.pressAmountFiveButton();
+        await SendView.pressContinueButton();
+        await SendView.inputRecipientAddress(RECIPIENT_ADDRESS_MOCK);
+        await SendView.pressReviewButton();
+
+        await Assertions.expectElementToBeVisible(
+          Matchers.getElementByID(
+            GasFeeTokenSelectorIDs.SELECTED_GAS_FEE_TOKEN_ARROW,
+          ),
+        );
+        await TransactionConfirmView.tapGasFeeTokenPill();
+
+        await GasFeeTokenModal.checkAmountFiat('DAI', '$9.44');
```

Comment:
small nit: can these strings be put in a variable or object?

 
```suggestion
const usdcValues = {
  fiatAmount: '$3.62',
  tokenAmount: '1.23 USDC',
  balance: 'Bal: $14.71 USD',
};

// Usage
await GasFeeTokenModal.checkAmountFiat('USDC', usdcValues.fiatAmount);
await GasFeeTokenModal.checkAmountToken('USDC', usdcValues.tokenAmount);
await GasFeeTokenModal.checkBalance('USDC', usdcValues.balance);
```